tool_collections/gops/basecoverage/operation_filter.py

```diff
@@ -1,8 +1,5 @@
 # runs after the job (and after the default post-filter)
-import os
-from galaxy import eggs
-from galaxy import jobs
-from galaxy.tools.parameters import DataToolParameter
+from galaxy.tools.parameters.basic import DataToolParameter
```

Comment:
Especially if it is only imported here - I think this should be reverted. That means maybe someday we can refactor and breakup basic.py :).